src/pages/products/product-add/products-add.js

```diff
@@ -131,7 +134,19 @@ const ProductsAdd = () => {
   return (
     <Paper className={styles.container}>
       <CustomizedStepper steps={steps} activeStep={activeStep} />
+      <div className={styles.controlsBlock}>
+        <Link to={config.routes.pathToProducts}>
+          <StandardButton
+              id='back'
+              title='Назад'
+              variant='outlined'
+              onClickHandler={() => {}}
+              data-cy='back-btn'
+          />
+        </Link>
```

Comment:
Please move title to config, and set link to onclickHandler

Reply:
done